api/v1/routes/newsletter_router.py

```diff
@@ -0,0 +1,67 @@
+from fastapi import (
+    APIRouter,
+    HTTPException,
+    Request,
+    Depends,
+    status
+    )
+from fastapi.responses import JSONResponse
+from sqlalchemy.orm import Session
+from api.v1.models.newsletter import NEWSLETTER
+from api.v1.schemas.newsletter_schema import EMAILSCHEMA
+from api.db.database import get_db, Base, engine
+
+
+Base.metadata.create_all(bind=engine)
+print('creating table')
```

Comment:
Can this be removed

Reply:
@bolexs I have removed the unnecessary Print statement used during development. Thank you for spotting that.